product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/src/main/synapse-config/proxy-services/1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<proxy name="1_6_1_4_Proxy_addPayloadStoredInPropertyAsChild" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+    <target>
+        <inSequence>
+            <property name="employeeInfo" scope="default">
+                <employeeInfo xmlns="">
+                    <firstName>Jacque</firstName>
+                    <lastName>Kallis</lastName>
+                    <team>EI</team>
+                </employeeInfo>
```

Comment:
Shall we make this element as follows:
<employeeInfo xmlns="">
  <employee>
      <firstName>Jacque</firstName>
      <lastName>Kallis</lastName>
      <team>EI</team>
  </employee>
</employeeInfo>

Reply:
This is done.